docs/mozillavpn.json

```diff
@@ -4,7 +4,7 @@
     "IOS": "2.11.0",
     "LINUX": "2.11.0",
     "MACOS": "2.11.0",
```

Comment:
do we want to say 2.11.1 here too?

Reply:
So, it look like when releng ran their publishing task, it pushed the 2.11.1 builds for both Windows and Mac to archive.mozilla.org. However, even though there ought to be no changes in the 2.11.1 build for MacOS we haven't actually done any testing on it.

I suppose we should go test what balrog does to see what happens on MacOS, and align this document with that.